codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/DafnyLocalServiceGenerator.java

```diff
@@ -208,33 +210,17 @@ func NewClient(clientConfig $L) (*$T, error) {
               SmithyNameResolver.smithyTypesNamespace(outputShape),
               outputShape.toShapeId().getName()
             );
-        String validationCheck = "";
```

Comment:
I have moved this code down because this (mostly return value) will change on positional trait.